addons/base_geolocalize/models/res_partner.py

```diff
@@ -60,18 +38,18 @@ def geo_query_address(street=None, zip=None, city=None, state=None, country=None
                                               country])))
 
 
-class res_partner(osv.osv):
+class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    _columns = {
-        'partner_latitude': fields.float('Geo Latitude', digits=(16, 5)),
-        'partner_longitude': fields.float('Geo Longitude', digits=(16, 5)),
-        'date_localization': fields.date('Geo Localization Date'),
-    }
+    partner_latitude = fields.Float(string='Geo Latitude', digits=(16, 5))
+    partner_longitude = fields.Float(string='Geo Longitude', digits=(16, 5))
+    date_localization = fields.Date(string='Geo Localization Date')
 
-    def geo_localize(self, cr, uid, ids, context=None):
-        # Don't pass context to browse()! We need country names in english below
-        for partner in self.browse(cr, uid, ids):
+    @api.multi
+    def geo_localize(self):
+        # Pass context in self! We need country names in english below
+        self = self.with_context(lang='en_US')
```

Comment:
That's bloody terrifying. Why rebind self instead of just iterating on `self.with_context(whatever)`?

Also semantics change, the old code used an empty/absent context, the new one just replaces the lang.